src/baseline_system/clustering_common.py

```diff
@@ -25,3 +25,34 @@ def cluster_mentions(mention_list, score):
             clusters.append(set([mention]))
 
     return clusters
+
+def cluster_mentions_with_max_cluster(mention_list, score):
+    """

```

Comment:
Where is this function coming from? Hitesh?

Reply:
No, I created it in order to examine some changes in clustering algorithm.